apps/stable_diffusion/src/utils/stable_args.py

```diff
@@ -476,6 +476,13 @@ def is_valid_file(arg):
     help="Specify target triple for metal.",
 )
 
+p.add_argument(
+    "--iree-vulkan-vma-allocator",
+    default=True,
```

Comment:
OK. I have default set to False.
I suppose the IREE flag will be removed so we don't need this patch.

Reply:
If we want it for a green CI we can disable by default and call it for SD testing temporarily. won't touch llamas and we have the option in the app until fixes are in.

Or we can keep it in a branch. I'll run the CI on it just to see if it will clear all issues after the tokenizers fix.